Dockerfile

```
@@ -51,8 +53,8 @@ RUN apt-get install -y python rebar
 
 # install and update python-pip
 RUN apt-get install -y python-pip python3-pip && \
-    pip2 install --upgrade pip==$PIP_INSTALL_VERSION  && \
-    pip3 install --upgrade pip==$PIP3_INSTALL_VERSION
+    pip2 install --no-cache --upgrade pip==$PIP_INSTALL_VERSION  && \
+    pip3 install --no-cache --upgrade pip==$PIP3_INSTALL_VERSION
```

@bluefish6 Did you mean `--no-cache-dir`? We were looking at available options for pip and we couldn't see `--no-cache`.

---

You're right of course, I got the options wrong :)

```suggestion
    pip2 install --no-cache-dir --upgrade pip==$PIP_INSTALL_VERSION  && \
    pip3 install --no-cache-dir --upgrade pip==$PIP3_INSTALL_VERSION
```